City-of-Helsinki/drupal-hdbt
# UHF-2519: Back to top button

How to test:

1. Take any environment you have set up or set one up from scratch.
2. `composer require  drupal/hdbt:dev-UHF-2519_footer_changes && composer require drupal/helfi_platform_config:dev-UHF-2519_footer_changes`
3. `make drush-cr`
4. `make drush-locale-update`
5. Go to the local instance that you have running and check that there is now a link on the footer that says "Back to top" (if you are viewing the site in Finnish "Takaisin ylös" and if in Swedish "Tillbaka till toppen")
6. Clicking this link should take you back to the top of the page.
7. The link should have focus borders around it when you take focus to it.
8. The link should look as it does in the layouts here: https://share.goabstract.com/a59a03aa-9862-4067-acda-8d17511773ed?collectionLayerId=4b1d79e9-7ce2-4c0a-a08f-873350251275&mode=design&sha=latest

Also check these:
https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/191
